packages/account/src/Helpers/utils.tsx

```diff
@@ -190,3 +192,40 @@ export const flatten = <T extends Array<unknown>>(arr: T) => [].concat(...arr);
 
 export const isServerError = (error: unknown): error is TServerError =>
     typeof error === 'object' && error !== null && 'code' in error;
+
+/**
+ *  Returns the alpha 3 code for a given country code
+ * @name convertAlpha2toAlpha3
+ * @param country_code  - country code
+ * @returns alpha 3 code
+ */
+export const convertAlpha2toAlpha3 = (country_code: string) =>
+    country_code.length !== 3 ? countries.alpha2ToAlpha3(country_code.toUpperCase()) : country_code;
+
+/**
+ * Returns the alpha 2 code for a given country code
+ * @name convertAlpha3toAlpha2
+ * @param country_code - country code
+ * @returns alpha 2 code
+ */
+
+export const convertAlpha3toAlpha2 = (country_code: string) =>
+    country_code.length !== 2 ? countries.alpha3ToAlpha2(country_code.toUpperCase()) : country_code;
+
+/**
+ * Generates a language code supported by Onfido
+ * @name getOnfidoSupportedLocaleCode
+ * @param language_code
+ * @returns language code supported by Onfido
+ */
+export const getOnfidoSupportedLocaleCode = (language_code: string) => {
+    try {
+        const code = language_code.toLowerCase().split('_');
+        if (code[0] === 'id') {
+            return LANGUAGE_CODES.ID;
+        }
+        return code.length > 1 ? `${code[0]}_${code[1].toUpperCase()}` : code[0];
```

Comment:
Why are we checking if `code.length > 1`, can a language code be single character?

Reply:
language_code can be 'xx' or 'xx_XX'. so after splitting it the variable 'code' can be ['xx'] or ['xx', 'XX']